src/protagonist/API/Infrastructure/Messaging/AssetNotificationSender.cs

```diff
@@ -101,11 +100,15 @@ private async Task<CustomerPathElement> GetCustomerPathElement(int customer)
         return customerPathElement;
     }
     
-    private async Task<bool> SendCleanupAssetRequest(IList<string> change, CancellationToken cancellationToken)
+    private async Task<bool> SendAssetModifiedRequest(Dictionary<ChangeType, List<string>> change, CancellationToken cancellationToken)
     {
         if (change.IsNullOrEmpty()) return true;
 
-        var toSend = change.Select(n => new AssetModifiedNotification(n, ChangeType.Delete)).ToList();
+        var toSend = (

```

Comment:
Why use query syntax here? I think this is the only place in the codebase where query syntax is used rather than method syntax